ios/FluentUI/Table View/TableViewCell.swift

```diff
@@ -1380,6 +1375,8 @@ open class TableViewCell: UITableViewCell {
     }
 
     private func updateAccessibility() {
+        accessibilityTraits.insert(.button)
```

Comment:
is that a fair assumption? what if client is using this that isn't actionable tableviewcell? shouldn't client have more say what accessible trait it should be?

Reply:
Initially I thought of conditioning this trait with the **isUserInteractionEnabled** property which is the scenario where the cell does not represent an action of any sort.

But if you look at the [isEnabled property](https://github.com/microsoft/fluentui-apple/blob/78aa5434e23d4c84333a3c27a9c9a60dc133307f/ios/FluentUI/Table%20View/TableViewCell.swift#L708), it sets isUserInteractionEnabled for a disabled button (that still needs to be treated as a button.